cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/etl/api/batch/BatchContext.java

```diff
@@ -29,6 +31,19 @@
 public interface BatchContext extends DatasetContext, TransformContext {
 
   /**
+   * Creates a dataset identified by datasetName, typeName with properties.
+   * See {@link co.cask.cdap.api.Admin} for details.
```

Comment:
For Javadoc comments, since it's delegating to Admin's createDataset and datasetExists, is it bad to copy-paste the javadoc for those methods?